examples/react-full/pages/star-wars/index/index.page.server.ts

```diff
@@ -1,11 +1,16 @@
 import fetch from 'node-fetch'
+import { withTypescript } from 'vite-plugin-ssr'
 import { filterMovieData } from '../filterMovieData'
 import type { Movie, MovieDetails } from '../types'
 
 export { onBeforeRender }
 export { prerender }
 
-async function onBeforeRender() {
+const onBeforeRender = withTypescript('onBeforeRender', async (pageContext) => {
+
+  // `documentProps?: { title: string }` is missing
+  pageContext.pageExports.documentProps?.title
```

Comment:
Doesn't seem to work? TS complains that `pageContext.pageExports.documentProps` is of type `unknown`.

Reply:
IDEM